pcsx2/System/SysCoreThread.cpp

```diff
@@ -316,7 +316,8 @@ void SysCoreThread::OnResumeInThread(bool isSuspended)
 		DoCDVDopen();
 	FWopen();
 	SPU2open((void*)pDsp);
-	DEV9open((void*)pDsp);
+	if (isSuspended)
+		DEV9open((void*)pDsp);
```

Comment:
We already have an `if (isSuspended)` for CDVD, wouldn't it make more sense to move everything there?

Reply:
I know that plug-ins where always suspended/ resumed in a specific order (for the most part)
I think this was only important for plug-ins that where subclassing the GSDX window (which I think lilypad & my usb plugin did) 

I pretty sure DEV9 would be unaffected, but I was keeping that convention

Reply:
If you deem it unimportant I can just move it to the above condition

Reply:
That's indeed a picky situation, but in our case we want critical plugins for IO, eg CDVD, to open before plugins that read from said IO, eg SPU2/GS.
You basically always want CDVD and Pad to be initialized ASAP, then initialize non critical plugins then at the end SPU2 and GS, probably.

DEV9 doesn't matter either way afaik, you can probably test it to make sure though.

Reply:
Additionally, I can also move DEV9 in OnSuspendInThread to preserve that convention

Reply:
It would be appreciated, thanks!